From 7e1b2369189fec0447aa55fb7ccd284dda5b6e85 Mon Sep 17 00:00:00 2001
From: kalicyh <34980061+kaliCYH@users.noreply.github.com>
Date: Tue, 21 Jan 2025 01:29:29 +0800
Subject: [PATCH 13/16] =?UTF-8?q?=F0=9F=A6=84=20refactor(APP):=20U2F=20?=
 =?UTF-8?q?=E6=B1=89=E5=8C=96?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 applications/main/u2f/views/u2f_view.c | 18 ++++++++----------
 1 file changed, 8 insertions(+), 10 deletions(-)

diff --git a/applications/main/u2f/views/u2f_view.c b/applications/main/u2f/views/u2f_view.c
index 3b905f640..daf3b351a 100644
--- a/applications/main/u2f/views/u2f_view.c
+++ b/applications/main/u2f/views/u2f_view.c
@@ -21,26 +21,24 @@ static void u2f_view_draw_callback(Canvas* canvas, void* _model) {
 
     if(model->display_msg == U2fMsgNotConnected) {
         canvas_draw_icon(canvas, 22, 15, &I_Connect_me_62x31);
-        canvas_draw_str_aligned(canvas, 128 / 2, 3, AlignCenter, AlignTop, "Connect to a device");
+        canvas_draw_str_aligned(canvas, 128 / 2, 3, AlignCenter, AlignTop, "连接到设备");
     } else if(model->display_msg == U2fMsgIdle) {
         canvas_draw_icon(canvas, 22, 15, &I_Connected_62x31);
-        canvas_draw_str_aligned(canvas, 128 / 2, 3, AlignCenter, AlignTop, "Connected!");
+        canvas_draw_str_aligned(canvas, 128 / 2, 3, AlignCenter, AlignTop, "已连接！");
     } else if(model->display_msg == U2fMsgRegister) {
-        elements_button_center(canvas, "OK");
+        elements_button_center(canvas, "确定");
         canvas_draw_icon(canvas, 22, 15, &I_Auth_62x31);
-        canvas_draw_str_aligned(canvas, 128 / 2, 3, AlignCenter, AlignTop, "Press OK to register");
+        canvas_draw_str_aligned(canvas, 128 / 2, 3, AlignCenter, AlignTop, "按确定进行注册");
     } else if(model->display_msg == U2fMsgAuth) {
-        elements_button_center(canvas, "OK");
+        elements_button_center(canvas, "确定");
         canvas_draw_icon(canvas, 22, 15, &I_Auth_62x31);
-        canvas_draw_str_aligned(
-            canvas, 128 / 2, 3, AlignCenter, AlignTop, "Press OK to authenticate");
+        canvas_draw_str_aligned(canvas, 128 / 2, 3, AlignCenter, AlignTop, "按确定进行认证");
     } else if(model->display_msg == U2fMsgSuccess) {
         canvas_draw_icon(canvas, 22, 15, &I_Connected_62x31);
-        canvas_draw_str_aligned(
-            canvas, 128 / 2, 3, AlignCenter, AlignTop, "Authentication successful!");
+        canvas_draw_str_aligned(canvas, 128 / 2, 3, AlignCenter, AlignTop, "认证成功！");
     } else if(model->display_msg == U2fMsgError) {
         canvas_draw_icon(canvas, 22, 15, &I_Error_62x31);
-        canvas_draw_str_aligned(canvas, 128 / 2, 3, AlignCenter, AlignTop, "Certificate error");
+        canvas_draw_str_aligned(canvas, 128 / 2, 3, AlignCenter, AlignTop, "证书错误");
     }
 }
 
-- 
2.39.5 (Apple Git-154)

